app/services/admin.py

```diff
@@ -9,7 +10,8 @@ def add_manager(data):
         existing_user = session.query(User).filter(User.username == data['username']).first()
         if existing_user:
             return {'error': 'Username is already taken'}, 400
-        new_user = User(username=data['username'], password=data['password'], email= data['email'],role='manager')
+        hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
```

Comment:
Why we don't have a function for hashing the password, If you are using it at multiple places.
And we are making this query twice,